easy-to-use-api/klips-url-generator/src/App.tsx

```diff
@@ -130,13 +139,24 @@ const App: React.FC = () => {
   return (
     <div className='App'>
       <MapContext.Provider value={map}>
-        <ConfigProvider 
+        <ConfigProvider
         >
           <div className='output-wrapper'>
             <div className='header'>
               <img className='logo' src="https://www.klips-projekt.de/wp-content/uploads/2021/02/SAG_KLIPS-Logo_Jan21.png" alt="KLIPS Logo"></img>
               <h2>Widget URL Generator</h2>
             </div>
+            <p className='information-text'>
+              Konfigurieren Sie sich die URL zu unseren Widgets, indem Sie das nachstehende
+              Formular ausfüllen. Mit der URL können Sie die aktuellen Daten aus dem KLIPS
+              Projekt einfach in Ihrer Webseite darstellen.</p><br />
+            <p className='information-text-bold'><b>Und so einfach geht's:</b></p><br />
+            <ol className='information-table'>
+              <li>Widget auswählen</li>
+              <li>URL generieren</li>
+              <li>Code-Beispiel in die Webseite integrieren</li>
+            </ol>
+            <p className='information-text'>Dabei aktualisieren sich die Wetterdaten regelmäßig von selbst!</p><br />
```

Comment:
Maybe we could define the interval here? (regelmäßig = stündlich)

Reply:
Good point that you! I have adjusted the wording.